runtime/stdlib/builtin.go

```diff
@@ -344,12 +215,67 @@ func BuiltinValues() StandardLibraryValues {
 			)
 		},
 		Kind: common.DeclarationKindEnum,
-	}
-
-	return StandardLibraryValues{
-		signatureAlgorithmValue,
-		hashAlgorithmValue,
-	}
+	},
+	{
+		Name: "BLS",
```

Comment:
@ramtinms If you are curious, this demonstrates how to declare a built-in `BLS` contract. We would define the RLP contract in a similar fashion